pyiron_base/interfaces/has_hdf.py

```diff
@@ -39,7 +40,7 @@ def __exit__(self, *args):
             self._hdf.close()
 
 
-class HasHDF(ABC):
+class HasHDF(HasDict, ABC):
```

Comment:
This then actually also needs to implement the necessary methods for the interface.  Just leave the subclass out here, if you don't want to do this now.

Reply:
```suggestion
class HasHDF(ABC):
```